tests/php/Security/MemberCsvBulkLoaderTest.php

```diff
@@ -9,6 +9,7 @@
 use SilverStripe\Security\PasswordValidator;
 use SilverStripe\Security\Security;
 use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\MemberAuthenticator\MemberAuthenticator;
```

Comment:
Could we use alphabetical order?

Reply:
This new import is needed in testCleartextPasswordsAreHashedWithDefaultAlgo()